LibertyDSNP/spec
# DIP-228 Remove Graph Change Announcement

Problem
=======
Graph Change Announcement is too limited and will be replaced with mutable state management operations. See DIP #228.

Solution
========
This PR removes references to Graph Change Announcement. The new solution will be addressed in a separate PR.

Change summary:
---------------
- [ ] Removed Graph Change Announcement page
- [ ] Removed link from navigation section
- [ ] Removed from Announcement Type table
- [ ] Added to pending spec changes
- [ ] Removed a reference in Frequency docs that was now a broken link. (Note, however, that this PR does not attempt to update Frequency documentation.)

Steps to Verify:
----------------
1. Review content changes.
2. Generate site (`npm run build`) and check changed pages.
